utils/data_manager.py

```
@@ -1,37 +1,40 @@
 """
 Data Manager module for handling session data storage and retrieval
 """
+
 import os
 import json
 import time
 from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)